RTNeural/gru/gru_eigen.h

```diff
@@ -193,16 +274,27 @@ class GRULayerT
     inline std::enable_if_t<srCorr == SampleRateCorrectionMode::None, void>
     computeOutput() noexcept
     {
-        outs = (out_type::Ones() - zVec).cwiseProduct(cVec) + zVec.cwiseProduct(outs);
+        for (int i = 0; i < out_sizet; ++i)
+        {
+            outs(i) = extendedHt1(i);
+        }
```

Comment:
Similarly, I think there should be a way to avoid this copy... I'll try out a couple of ideas.